src/plugins/plot/spectralAggregatePlot/SpectralAggregateView.vue

```diff
@@ -21,15 +21,314 @@
 -->
 
 <template>
-<div>
-
+<div class="c-spectral-aggregate-plot-view gl-plot plot-legend-bottom"
+     :class="{'plot-legend-expanded': legendExpanded, 'plot-legend-collapsed': !legendExpanded }"
+>
+    <SpectralAggregatePlot ref="spectralAggregatePlot"
+                           class="c-spectral-aggregate-plot__plot-wrapper"
+                           :data="visibleData"
+                           :plot-axis-title="plotAxisTitle"
+                           :legend-expanded="legendExpanded"
+    />
 </div>
 </template>
 
 <script>
+import * as SPECTRAL_AGGREGATE from './SpectralAggregateConstants';
+import ColorPalette from '../lib/ColorPalette';
+import objectUtils from 'objectUtils';
```

Comment:
Instead of importing objectUtils here, you can use the ObjectAPI - https://github.com/nasa/openmct/blob/mct4102/src/api/objects/ObjectAPI.js#L458